apps/backend/src/tests/neighborhood_api.test.ts

```diff
@@ -159,11 +164,11 @@ describe('Tests for creating a single neighborhood: POST /neighborhoods/:id ', (
   // error was 'socket hang up'
   test('when user not logged in, unable to create neighborhood', async () => {
     const postResponse: Response = await api.post('/api/neighborhoods');
-    // console.log(postResponse?.body);
+    console.log(postResponse?.body);
```

Comment:
Do we still need this? Same for line 171

Reply:
Not sure. It's been a while since I got this error, but because it's untraceable when it happens, I've left these to help with debugging if it occurs again.